source/render/ODFrameListener.cpp

```diff
@@ -92,8 +92,6 @@ ODFrameListener::ODFrameListener(Ogre::RenderWindow* renderWindow, Ogre::Overlay
 
     mGameMap = new GameMap(false);
 
-    mMiniMap = new MiniMap(mGameMap);
```

Comment:
I've already run a profiler and filling the minimap was the most time consuming back then (before @Bertram25 's PR). I've not tried later.